addons/mail/models/mail_channel.py

```diff
@@ -1204,7 +1204,7 @@ def execute_command_help(self, **kwargs):
         else:
             all_channel_partners = self.env['mail.channel.partner'].with_context(active_test=False)
             channel_partners = all_channel_partners.search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)])
-            msg = _("You are in a private conversation with <b>@%s</b>.", html_escape(channel_partners[0].partner_id.name if channel_partners else _('Anonymous')))
+            msg = _("You are in a private conversation with <b>@%s</b>.", html_escape(channel_partners[0].partner_id.name if channel_partners else self.env.user.partner_id.name))
```

Comment:
Hello, same remarks as enterprise PR in community. Also we could wonder why is the code for this part is duplicated in 2 different places.